app/helpers/MetaFormats/FormatDefinitions/GroupPrivateDataFormat.php

```diff
@@ -21,51 +21,51 @@ class GroupPrivateDataFormat extends MetaFormat
     #[FPost(new VArray(new VUuid()), "IDs of all users that have admin privileges to this group (including inherited)")]
     public array $admins;
 
-    #[FPost(new VArray(new VUuid()))]
+    #[FPost(new VArray(new VUuid()), "IDs of all group supervisors")]
     public array $supervisors;
 
-    #[FPost(new VArray(new VUuid()))]
+    #[FPost(new VArray(new VUuid()), "IDs of all group observers")]
     public array $observers;
 
-    #[FPost(new VArray(new VUuid()))]
+    #[FPost(new VArray(new VUuid()), "IDs of the students of this group")]
     public array $students;
 
-    #[FPost(new VUuid(), required: false)]
+    #[FPost(new VUuid(), "The instance ID of the group", required: false)]
     public string $instanceId;
 
-    #[FPost(new VBool())]
+    #[FPost(new VBool(), "Whether the group has a valid license")]
     public bool $hasValidLicence;
 
-    #[FPost(new VArray(new VUuid()))]
+    #[FPost(new VArray(new VUuid()), "IDs of all group assignments")]
     public array $assignments;
 
-    #[FPost(new VArray(new VUuid()))]
+    #[FPost(new VArray(new VUuid()), "IDs of all group shadow assignments")]
     public array $shadowAssignments;
 
-    #[FPost(new VBool())]
+    #[FPost(new VBool(), "Whether the group statistics are public")]
     public bool $publicStats;
 
-    #[FPost(new VBool())]
+    #[FPost(new VBool(), "Whether the group is detaining")]
     public bool $detaining;
 
-    #[FPost(new VDouble(), required: false)]
+    #[FPost(new VDouble(), "The group assignment point threshold", required: false)]
```

Comment:
A relative number of points a student must receive from assignments to fulfill the requirements of the group.